src/wram.asm

```diff
@@ -1475,7 +1569,13 @@ wcecc:: ; cecc
 wcece:: ; cece
 	ds $2
 
-	ds $47
+	ds $a
+
+; pointer to memory to store AI temporary hand card list
+wHandTempList:: ; ceda
```

Comment:
This doesn't look like a pointer, but rather a buffer for a card list similar to ``wDuelTempList``? I would guess they have the same size too.

Reply:
Yeah, it is indeed a buffer, I just had a brain fart there, heh.